src/builtins.c

```diff
@@ -39,6 +39,8 @@ DLLEXPORT void NORETURN jl_error(const char *str)
     jl_throw(jl_new_struct(jl_errorexception_type, msg));
 }
 
+extern int vasprintf(char **str, const char *fmt, va_list ap);
```

Comment:
Don't think so, we have our own at `src/support/asprintf.c` that we use elsewhere, just not declared in this file yet.